### PR TITLE
239 airview documentation

### PR DESCRIPTION
**Pull request by BinaryJim**
Adds documentation for:

- `airview root repo`
- `airview-cms`
- `airview-mock-server`
- `airview-ui`

closes airwalk-digital/airview-issues#239

### CODE REVIEW

File: packages/airview-cms/README.md [hunk 1]
```diff
@@ -1 +1,395 @@
...
+```javascript
+{
+  label: "Related Coffee",
+  name: "related_coffee",
+  widget: "entrySelect",
+  collection: "coffee"
+}
+```
+
+## CMS Context
```
Comment (mcuckson) [inline packages/airview-cms/README.md:225]: Does JS/React have a way of documenting the below hooks in code for hinting in the IDE? The documentation is good, but it'd be helpful to have it with the code. I know we can generate this type of documentation from code in other languages. Given time constraints, we should probably get this merged the raise a task to cover moving it into code later.

**Review by mcuckson — CHANGES_REQUESTED** [review]
Just a few comments and let's raise issues for the bits needed and we'll merge this as is.

File: packages/airview-cms/README.md (hunk 1 above)
Comment (BinaryJim) [reply packages/airview-cms/README.md:225]: Good question, I'm not sure to be honest. 

Moving to TypeScript would help, but that's a big task; JSDoc could work, failing that we could inline an example in the ReadMe.

Added a ticket to track:

https://github.com/AirWalk-Digital/airview-issues/issues/259

File: README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+A demonstration of the integration of `airview-ui`, `airview-cms`, `airview-mock-server` and `airview-cms-api` packages. The demo acts as a development environment and an ideal canvas to talk through features or issues, without having to hit a real GitHub repo or GitHub API. [Learn more](https://github.com/AirWalk-Digital/airview/tree/main/apps/airview-demo)
+
+### Airview API Demo
+
+[INSERT DESCRIPTION HERE]
+
+## Cloning the monorepo for local development
+
+The monorepo uses [npm workspaces](https://docs.npmjs.com/cli/v8/using-npm/workspaces). A requirement for the project is node.js LTS, currently greater that **version 16.15.1** but less than **version 17.0.0**.
```
[the remark this answers is not in the document]
Comment (BinaryJim) [reply README.md:41]: Good shout, ticket added to track:

https://github.com/AirWalk-Digital/airview-issues/issues/260

File: README.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+Branch names should include the GitHub issue ticket number, as should all commits. When a pull request is merged into main, please ensure you delete the branch from remote.
+
+### Pull requests
+
+Our `main` branch is protected; any pull request to `main` will require the review and approval from an authorised reviewer, before a merge can be made.
+
+There is no specific format for pull request information, we would ask that you provide a brief description of the work and use [closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) to automate the closure of the related ticket in GitHub issues.
+
+### Versioning releases
```
[the remark this answers is not in the document]
Comment (BinaryJim) [reply README.md:67]: Thanks for this, updated the readme with the above contents

File: packages/airview-cms/README.md [hunk 4]
```diff
@@ -1 +1,395 @@
...
+1. A WYSIWYG editing experience, which allows the creation and editing of Markdown documentation (body and frontmatter)
+2. A simplified interface to allow the persistence of markdown content using a gitflow workflow (commit, branch and checkout)
+
+**For the engineer, Airview CMS is:**
+
+1. A flexible React CMS, it can suit the needs of any project via a provided configuration
+2. A set of React hooks, these simplify the request of CMS data from the remote repository
+3. A React components for inline, real-time WYSIWYG markdown editing
+
+### Collections
```
[the remark this answers is not in the document]
Comment (BinaryJim) [reply packages/airview-cms/README.md:18]: Good shout, another argument for using TypeScript eh! ;)

Ticket added to track:

https://github.com/AirWalk-Digital/airview-issues/issues/261

**Review by mcuckson — APPROVED** [review]

**Pull request merged**